src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,33 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    static final String SEARCH_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> list = new ArrayList<String>();
+        ArrayList<String> resultList = new ArrayList<String>();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            while (reader.ready()) {
+                list.add(reader.readLine());
+            }
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
```

Comment:
the logic is mixed here
let's catch both Exceptions in one catch clause using pipe `|`
and rethrow them as a RuntimeException

Reply:
done

Reply:
I can't do this 2 exseption with a pipe, beacause of message 'catch' child has incorrect indentation level 11, expected level should be 12. [Indentation]
Please help with that